howdy/src/meson.build

```diff
@@ -2,13 +2,14 @@ if meson.is_subproject()
 project('howdy', 'cpp', license: 'MIT', version: 'beta', meson_version: '>= 0.64.0')
 endif
 
-py = import('python').find_installation()
-py.dependency()
 
 datadir = get_option('prefix') / get_option('datadir') / 'howdy'
 py_conf = configuration_data(paths_dict)
 py_conf.set('data_dir', datadir)
 
+py = import('python').find_installation(paths_dict.get('python_path'))
```

Comment:
Not sure about this, python_path could be another path that the packager would set?